src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ExampleTests.java

```diff
@@ -0,0 +1,95 @@
+
+import org.joda.time.DateTime;
+import org.openqa.selenium.TimeoutException;
+import org.testng.annotations.Test;
+
+@Test(groups = "ExampleTests")
+public class ExampleTests extends NewTestTemplate {
+
+  @Test
+  public void articlePushedWithAPIHasItsContent() {
+    // push special article through API
+    String specialText = "!!!</html>";
+    String articleContentSent = PageContent.ARTICLE_TEXT + specialText;
+    new ArticleContent().push(articleContentSent);
+
+    // get article
+    ArticlePageObject article = new ArticlePageObject().open();
+    String articleContentReceived = article.getContent();
+
+    // check if received content contains special text
+    Assertion.assertStringContains(articleContentReceived, specialText);
+  }
+
+  @Test
+  @Execute(asUser = User.USER)
+  public void articleAddedContentClassicEditor() {
+    // create a new article
+    new ArticleContent().push("");
+
+    // open the article and enter the classic editor
+    ArticlePageObject article = new ArticlePageObject().open();
+    VisualEditModePageObject classicVisualEditMode = article.openCKModeWithMainEditButtonDropdown();
+
+    // add text and photo content
+    classicVisualEditMode.addContent(PageContent.ARTICLE_TEXT);
+    PhotoAddComponentObject photoAddPhoto = classicVisualEditMode.clickPhotoButton();
+    PhotoOptionsComponentObject photoOptions = photoAddPhoto.addPhotoFromWiki("image", 1);
+    photoOptions.setCaption(PageContent.CAPTION);
+    photoOptions.clickAddPhoto();
+    classicVisualEditMode.clickPublishButton();
+
+    // verify photo and text
+    article.refreshPage();
+    Assertion.assertStringContains(article.getArticleTextRaw(), PageContent.ARTICLE_TEXT);
+    article.verifyPhoto();
+  }
+
+  @Test
+  @Execute(asUser = User.USER_9)
+  public void articleLinkingToOther() {
+    String linkingArticleTitle = PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis() +
+                               "LinkingArticle";
+    String linkedToArticleTitle = PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis() +
+                               "LinkedToArticle";
+    // create an article with real content
+    new ArticleContent().push(PageContent.ARTICLE_TEXT_DROZDY, linkedToArticleTitle);
+
+
+    // get the link to the content article and add it in a linking article
+    new ArticlePageObject().open(linkedToArticleTitle).waitForPageLoad();
+    String linkToArticle = new ArticlePageObject().getCurrentUrl();
+    new ArticleContent().push("["+linkToArticle+"]", linkingArticleTitle);
+
+    // open the newly created article with a link
+    new ArticlePageObject().open(linkingArticleTitle);
+
+    // click the link and verify we're back on the content article
+    // due to Expected condition failed: waiting for url to contain... sometimes
+    // don't see other way to suppress it
+    try {
+      new ArticlePage().clickArticleLink(0);
+    } catch (TimeoutException timeoutException){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00108)